### ISSUE TITLE
websocket issue with new colibri websocket regex scheme in nginx

### ISSUE THREAD

**Issue by jmallach** [jitsi/docker-jitsi-meet#1650]
Hi!

After upgrading to the latest release which includes https://github.com/jitsi/docker-jitsi-meet/commit/825730d6597a92b4f06c14eaf54a45e0d4667527, we started getting websocket connection issues between our rproxy and jvb.

Reverting to `ENABLE_COLIBRI_WEBSOCKET_UNSAFE_REGEX` fixes the issue.

In our setup, Jitsi is a docker compose stack behind a Traefik rproxy. The stack has worked as expected until this change.

We have the default jitsi.meet internal network as well as an external one for communication with Traefik and the internet.

The lookup for `jvb` resolves to the external network, but the websocket connection urls had `meet.jitsi` network internal IPs instead, so connections were refused.

Let me know if you need additional information. Thanks!

**Comment by saghul** [comment]
You can enable that, that's why it's an option. It does serve as a precaution for not allowing connections to arbitrary IPs via the proxy.

**Comment by jmallach** [comment]
Is there no way to allow for all the docker networks that the container is part of?

**Comment by jmallach** [comment]
Right, I can do that. I imagine many people will stumble on the new behaviour though, I suggest adding a pointer in the docs.

Thanks!

**Comment by aaronkvanmeerten** [comment]
I fully acknowledge that this change impacts many users, depending on how they have their systems configured. The problem was that the previous set up was so insecure as to be unacceptable/actually being used for attacks. we did a small announcement on our community and have updated the handbook. I am sorry that this impacted you this way, and we will work to improve our messaging when changes like this come in.

**Comment by xoxys** [comment]
Fully agree and nothing against this change or breaking changes in general. Just wanted to kindly ask for a better announcement of breaking changes in the future. Keep up your good work on Jitsi.
